src/bin/ion/commands/beta/generate/utils.rs

```diff
@@ -76,25 +171,15 @@ pub enum Template {
     Struct, // Represents a template for a Rust struct or Java class
 }
 
-impl Template {
-    /// Returns a string that represent the template file name based on given programming language.
-    pub fn name(&self, language: &Language) -> &str {
```

Comment:
(🗺️ PR tour)This has been moved on to `template_as_string()` in `JavaLanguage` and `RustLanguage`.